### PR TITLE
find: add pt_BR translation

### PR DESCRIPTION
**Pull request by dnielMantov**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 4.9.0

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.pt_BR/common/find.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# find
+
+> Procura recursivamente por arquivos ou diretórios em uma árvore de diretórios.
+> Mais informações: <https://manned.org/find>.
+
+- Procura por arquivos pela extensão:
+
+`find {{caminho_raiz}} -name '{{*.ext}}'`
+
+- Procura por arquivos que correspondem a padrões de caminho ou nome específico:
```
Comment (vitorhcl) [inline pages.pt_BR/common/find.md:10]:
```suggestion
- Procura por arquivos que correspondam a padrões de caminho ou nome específico:
```

File: pages.pt_BR/common/find.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`find {{caminho_raiz}} -path '{{**/caminho/**/*.ext}}' -or -name '{{*nome*}}'`
+
+- Procura por diretórios que correspondem a um nome específico, sem o uso de case-sensitive:
```
Comment (vitorhcl) [inline pages.pt_BR/common/find.md:14]:
```suggestion
- Procura por diretórios que correspondam a um nome específico, sem o uso de case-sensitive:
```

File: pages.pt_BR/common/find.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`find {{caminho_raiz}} -type d -iname '{{*nome*}}'`
+
+- Procura por arquivos que correspondem a um nome específico, excluindo certos caminhos:
```
Comment (vitorhcl) [inline pages.pt_BR/common/find.md:18]:
```suggestion
- Procura por arquivos que correspondam a um nome específico, excluindo certos caminhos:
```

File: pages.pt_BR/common/find.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`find {{caminho_raiz}} -name '{{*.py}}' -not -path '{{*/caminho/*}}'`
+
+- Procura por arquivos que correspondem a uma faixa de tamanho específica, limitando a profundidade recursiva para "1":
```
Comment (vitorhcl) [inline pages.pt_BR/common/find.md:22]:
```suggestion
- Procura por arquivos que correspondam a uma faixa de tamanho específica, limitando a profundidade recursiva para "1":
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]
Thank you, Daniel!

O verbo "corresponder" foi usado no indicativo, mas ele deve estar no subjuntivo, para especificar uma condição (se o arquivo corresponde)

LGTM after my suggestion

**Review by vitorhcl — APPROVED** [review]
Thank you, merging now 🚀

**Pull request merged**